editor/src/core/model/element-template-utils.ts

```diff
@@ -636,8 +655,14 @@ export function insertJSXElementChildren(
             projectContents,
             elementsToInsert.map((child) => child.uid),
           )
-          insertedChildrenPaths = elementsToInsert.map((child) =>
-            EP.appendToPath(EP.appendToPath(parentPath, newFragmentUid), child.uid),
+
+          insertedChildrenPaths = elementsToInsert.reduce(
```

Comment:
Looks like it would be worth extracting out this logic as it's repeated 3 times.

Reply:
The usages are very similar, but I think they are just different enough so that a generalised version wouldn't be much less complex (because the value added in the callback of the reduce is always different)